Error: Cannot find module '../../package.json'

- [x] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [x] I am using the latest version of `npm-check-updates`
- [x] I am using `node >= 10.17`

I'm using the ncu 12.0.1 and Node.js 17.0.1 and 16.13.0 both. 

---------------------------

## Steps to Reproduce

```sh
ncu
```

## Current Behavior

```sh
ledsun@MSI:~/textae►ncu
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '../../package.json'
Require stack:
- /home/ledsun/.local/share/nvm/v17.0.1/lib/node_modules/npm-check-updates/build/src/bin/cli.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/home/ledsun/.local/share/nvm/v17.0.1/lib/node_modules/npm-check-updates/build/src/bin/cli.js:11:40)
    at Module._compile (node:internal/modules/cjs/loader:1095:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1147:10)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/ledsun/.local/share/nvm/v17.0.1/lib/node_modules/npm-check-updates/build/src/bin/cli.js'
  ]
}

Node.js v17.0.1
```

## Expected Behavior

```sh
ledsun@MSI:~/textae►ncu
Checking /home/ledsun/textae/package.json
[====================] 48/48 100%

 ajv                         8.6.3  →   8.7.1
 @typescript-eslint/parser   5.3.0  →   5.3.1
 eslint                     7.32.0  →   8.2.0
 terser-webpack-plugin      ^5.2.4  →  ^5.2.5
 webpack                    5.61.0  →  5.62.1

Run ncu -u to upgrade package.json
```

I changed `"../../package.json"` to `"../../../package.json"` in `/home/ledsun/.local/share/nvm/v17.0.1/lib/node_modules/npm-check-updates/build/src/bin/cli.js:11`, then the ncu runs expected.

--- a/package.json
+++ b/package.json
@@ -128,7 +128,7 @@
     "yarn": "^1.22.17"
   },
   "files": [
-    "build/src",
+    "build",
     "!**/test/**"
   ],
   "lockfile-lint": {

